### PR TITLE
Standardize on NET pre-proc symbol

### PR DESCRIPTION
**Pull request by jaredpar**
A good chunk of our code uses `#if NETCOREAPP` instead of `#if NET` for historical reasons. Did a quick replace so we can standardize on the more idiomatic approach

### CODE REVIEW

**Comment by jaredpar** [comment]
@dotnet/roslyn-compiler PTAL
@CyrusNajmabadi, @ToddGrun standardizing from the question you all had yesterday

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
Not sure how i feel about code becoming more consistent...

File: src/Compilers/VisualBasic/Test/Semantic/SourceGeneration/GeneratorDriverTests.vb [hunk 1]
```diff
@@ -721,7 +721,7 @@ End Class
         End Sub
 
         Shared Sub VerifyArgumentExceptionDiagnostic(diagnostic As Diagnostic, generatorName As String, message As String, parameterName As String, Optional initialization As Boolean = False)
-#If NETCOREAPP Then
+#if NET Then
```
Comment (ToddGrun) [inline src/Compilers/VisualBasic/Test/Semantic/SourceGeneration/GeneratorDriverTests.vb:724]: >#if [](http://example.com/codeflow?start=0&length=3)

uber nit: inconsistent casing in vb files :)

**Review by ToddGrun — APPROVED** [review]
:shipit:

**Review by cston — APPROVED** [review]

**Comment by jaredpar** [comment]
@333fred, @chsienki, @jjonescz PTAL.

**Review by chsienki — APPROVED** [review]

**Pull request merged**